scripts/gen_binding.py

```diff
@@ -508,6 +508,7 @@ def normalize_arg(arg):
   const requires_stats = {js_requires_stats}
 
   let stats = null
```

Comment:
this remains unchanged right? I guess we can inline that into `t.stats = null` later in the code

Reply:
I kept the core stats logic as untouched as possible to keep scope down, but have ideas for cleanup/enhancements in follow up.